src/internal/keycache/cache.go

```diff
@@ -10,21 +10,20 @@ import (
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/backoff"
 	col "github.com/pachyderm/pachyderm/v2/src/internal/collection"
-	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/watch"
 )
 
 // Cache watches a key in etcd and caches the value in an atomic value
 // This is useful for frequently read but infrequently updated values
 type Cache struct {
-	c            col.Collection
+	c            col.EtcdCollection
```

Comment:
Is this because we don't have revisions in Postgres? What functionality is missing?

Reply:
This has been moved over to `PostgresCollection` in the `postgres-intergration` branch - this branch isn't supposed to move anything over to postgres collections yet.